processing/src/main/java/io/druid/segment/data/TmpFileIOPeon.java

```diff
@@ -34,8 +34,19 @@
 */
 public class TmpFileIOPeon implements IOPeon
 {
+  boolean allowConflict;
```

Comment:
this option is useful when we merge index, if we used lots of temp files and did not want then to conflict.
The original behavior is allow us to overwrite the tmp file, but it's dangerous sometimes.